spec/spec_helper.rb

```diff
@@ -3,7 +3,7 @@
 require "logstash/inputs/ganglia"
 require_relative "support/client"
 
-module GangliaHelpers
+class GangliaHelpers
 
   def setup_clients(number_of_clients, port)
     number_of_clients.times.inject([]) do |clients|
```

Comment:
I left it because I used in former test, but I agree we can removed now. we can have it here again if we need it.